core/trino-spi/src/main/java/io/trino/spi/connector/ConnectorCapabilities.java

```diff
@@ -16,4 +16,5 @@
 public enum ConnectorCapabilities
 {
     NOT_NULL_COLUMN_CONSTRAINT,
+    COMMENT_ON_TABLE,
```

Comment:
It should be add into ``MongoConnector`` after the pr #11424 merged

Reply:
per https://github.com/trinodb/trino/issues/11348#issuecomment-1061987450 i think it's too late to add this to ConnectorCapabilities. The damage has been done and the ship has sailed. Adding this now would be a breaking change.

cc @hashhar @ebyhr

Reply:
Agreed. Instead I'd suggest to add a test to BaseConnectorTest that asserts that COMMENT ON TABLE works.

For the connectors where it fails you need to disallow it in their ConnectorMetadata implementations.

Reply:
@findepi @hashhar Sorry, I'm a little busy these days so I can't reply in time.I don't understand why now is too late  to add this to ConnectorCapabilities. Is there some connectors is supported ``COMMENT ON TABLE`` but is not supported ``CREATE TABLE ... COMENT``.